app-admin/mkosi/mkosi-v10.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+
+DESCRIPTION="Build Bespoke OS Images"
```

Comment:
I found this description hard to comprehend. Maybe it is because I am not a native speaker. But I would suggest

```
DESCRIPTION="A tool for generating OS images"
```

Reply:
Yeah, I prefer this I think. You could do "generating (bespoke) OS images" if you really want.